### PR TITLE
Adjust documentation for AzureBlobStorage Databus v5 targetting Core v8

### PR DESCRIPTION
**Pull request by lailabougria**

### CODE REVIEW

File: nservicebus/upgrades/absdatabus-4to5.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+reviewed: 2020-10-29
+component: ABSDataBus
+related:
+ - nservicebus/upgrades/7to8
+isUpgradeGuide: true
+upgradeGuideCoreVersions:
+ - 8
+---
+
+Nothing to report...
```
Comment (lailabougria) [inline nservicebus/upgrades/absdatabus-4to5.md:12]: I know this needs some love here, but I wasn't sure what to add here. There will be meaningful changes from 3 to 4, but we don't have any changes as of now, between 4 to 5, except the impact created by Core v8. Thoughts?

File: Snippets/ABSDataBus/ABSDataBus_5/Usage.cs [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+            .Container("containerName");
+
+        #endregion
+    }
+    
+    #region CustomBlobServiceClientProvider
+
+    public class CustomProvider : IProvideBlobServiceClient
+    {
+        // Leverage dependency injection to get a preconfigured BlobServiceClient with the suitable configuration
```
Comment (SeanFeldman) [inline Snippets/ABSDataBus/ABSDataBus_5/Usage.cs:46]:
```suggestion
        // Leverage dependency injection to use a custom-configured BlobServiceClient
```

File: nservicebus/messaging/databus/azure-blob-storage_config_ABSDataBus_[5.0,).partial.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+The provider is then registered in the container:
+
+snippet: AzureDataBusInjectServiceClient
+
+#### Providing a connection string and container name
+
+snippet: AzureDataBusConnectionAndContainer
+
+NOTE: The container name is not required and will be set to a default value unless a value is supplied.
```
Comment (SeanFeldman) [inline nservicebus/messaging/databus/azure-blob-storage_config_ABSDataBus_[5.0,).partial.md:25]:
```suggestion
NOTE: The container name is optional and will be set to the default when omitted.
```

File: nservicebus/upgrades/absdatabus-4to5.md (hunk 1 above)
Comment (SeanFeldman) [reply nservicebus/upgrades/absdatabus-4to5.md:12]: Then we [keep it real](https://www.youtube.com/watch?v=20ob9meWOC4)
```suggestion
Upgrade from version 7 to 8 does not require any code changes.
```

File: samples/azure/blob-storage-databus/ABSDataBus_5/AzureBlobStorageDataBus.sln.DotSettings [hunk 4]
```diff
@@ -0,0 +1,6 @@
+<wpf:ResourceDictionary xml:space="preserve" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:s="clr-namespace:System;assembly=mscorlib" xmlns:ss="urn:shemas-jetbrains-com:settings-storage-xaml" xmlns:wpf="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
+	<s:String x:Key="/Default/Environment/InjectedLayers/FileInjectedLayer/=2EF370E98D1FDC4E9FD6E29F173EB613/RelativePath/@EntryValue">..\..\..\..\..\tools\Shared.DotSettings</s:String>
+	<s:Boolean x:Key="/Default/Environment/InjectedLayers/FileInjectedLayer/=2EF370E98D1FDC4E9FD6E29F173EB613/@KeyIndexDefined">True</s:Boolean>
+	<s:Boolean x:Key="/Default/Environment/InjectedLayers/InjectedLayerCustomization/=File2EF370E98D1FDC4E9FD6E29F173EB613/@KeyIndexDefined">True</s:Boolean>
+	<s:Double x:Key="/Default/Environment/InjectedLayers/InjectedLayerCustomization/=File2EF370E98D1FDC4E9FD6E29F173EB613/RelativePriority/@EntryValue">1</s:Double>
+	</wpf:ResourceDictionary>
```
Comment (SeanFeldman) [inline samples/azure/blob-storage-databus/ABSDataBus_5/AzureBlobStorageDataBus.sln.DotSettings:6]: Hmmm, what's this yummy stuff? 😄

**Review by SeanFeldman — APPROVED** [review]

File: nservicebus/upgrades/absdatabus-4to5.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+reviewed: 2020-10-29
+component: ABSDataBus
+related:
+ - nservicebus/upgrades/7to8
+isUpgradeGuide: true
+upgradeGuideCoreVersions:
+ - 8
+---
+
+Upgrade from version 7 to 8 does not require any code changes.
```
Comment (danielmarbach) [inline nservicebus/upgrades/absdatabus-4to5.md:12]:
```suggestion
Upgrade from version 4 to 5 does not require any code changes.
```

although that is probably soon no longer true

File: samples/azure/blob-storage-databus/ABSDataBus_5/Receiver/Receiver.csproj [hunk 6]
```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>netcoreapp3.1;netcoreapp2.1;net48</TargetFrameworks>
+    <OutputType>Exe</OutputType>
+    <LangVersion>7.3</LangVersion>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Azure.KeyVault.Core" Version="2.*" />
+    <PackageReference Include="Microsoft.Data.Services.Client" Version="5.*" />
+    <PackageReference Include="Newtonsoft.Json" Version="10.*" />
```
Comment (danielmarbach) [inline samples/azure/blob-storage-databus/ABSDataBus_5/Receiver/Receiver.csproj:10]: Pretty sure latest is 12

File: samples/azure/blob-storage-databus/ABSDataBus_5/Receiver/Receiver.csproj [hunk 7]
```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>netcoreapp3.1;netcoreapp2.1;net48</TargetFrameworks>
+    <OutputType>Exe</OutputType>
+    <LangVersion>7.3</LangVersion>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Azure.KeyVault.Core" Version="2.*" />
+    <PackageReference Include="Microsoft.Data.Services.Client" Version="5.*" />
```
Comment (danielmarbach) [inline samples/azure/blob-storage-databus/ABSDataBus_5/Receiver/Receiver.csproj:9]: Why are those needed?

File: samples/azure/blob-storage-databus/ABSDataBus_5/Sender/Sender.csproj [hunk 8]
```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>netcoreapp3.1;netcoreapp2.1;net48</TargetFrameworks>
+    <OutputType>Exe</OutputType>
+    <LangVersion>7.3</LangVersion>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Azure.KeyVault.Core" Version="2.*" />
```
Comment (danielmarbach) [inline samples/azure/blob-storage-databus/ABSDataBus_5/Sender/Sender.csproj:8]: Same comments as above

File: samples/azure/blob-storage-databus/ABSDataBus_5/Receiver/Receiver.csproj (hunk 7 above)
Comment (SeanFeldman) [reply samples/azure/blob-storage-databus/ABSDataBus_5/Receiver/Receiver.csproj:9]: Those are likely from WindowsAzure.Storage days.

File: nservicebus/upgrades/absdatabus-4to5.md (hunk 5 above)
Comment (lailabougria) [reply nservicebus/upgrades/absdatabus-4to5.md:12]: Are you suggesting to drop the cleanup in 5 but not 4?

Comment (SeanFeldman) [reply nservicebus/upgrades/absdatabus-4to5.md]: Deprecate with error in 4 and remove in 5.

**Comment by SeanFeldman** [comment]
Closing in favour of #5085
